tests/unit_tests/seraphis_serialization.cpp

```diff
@@ -210,7 +172,5 @@ TEST(seraphis_serialization_demo, seraphis_squashed_standard)
 
     EXPECT_TRUE(original_tx_id == recovered_tx_id);
     EXPECT_NO_THROW(EXPECT_TRUE(sp_tx_squashed_v1_size_bytes(tx) == sp_tx_squashed_v1_size_bytes(recovered_tx)));
-    EXPECT_TRUE(validate_tx(tx, tx_validation_context));
-    EXPECT_TRUE(validate_tx(recovered_tx, tx_validation_context));
```

Comment:
So it was removed because the number of bins couldn't be implied with mock txs, since there wasn't a static seraphis ref set config for all mock txs, but I fixed this problem by just adding in a `mock_num_bins` parameter to the serialize function with the latest commit